tasksJS/task1.js

```diff
@@ -0,0 +1,328 @@
+//----Strings----//
+
+//1 Remove String Spaces
+//https://www.codewars.com/kata/57eae20f5500ad98e50002c5
+
+function noSpace(x){
+    return x.split(" ").join("");
+}
+
+//2 Name Shuffler
+//https://www.codewars.com/kata/559ac78160f0be07c200005a 
+
+function nameShuffler(str){
+    return  str.split(" ").reverse().join(" ");
+}
+
+//3 Sum Mixed Array
+//https://www.codewars.com/kata/57eaeb9578748ff92a000009 
+
+function sumMix(x) {
+    let sum = 0;
+    for (let i = 0; i < x.length; i++) {
+        sum =sum + +x[i];
+    }
+    return sum;
+}
+
+//4 Multiplication table for number
+//https://www.codewars.com/kata/5a2fd38b55519ed98f0000ce
+
+//не могу найти недостающий пробел
+//30 минут
+function multiTable(number) {
+    let res = '';
+    for (let i = 1; i <= 10; i++) {
```

Comment:
it's better to move 10 to const variable, please avoid magic numbers

Reply:
what is magic number?let?)